### PR TITLE
Add more info about YAML config files.

### PR DESCRIPTION
**Pull request by vestimir**
As discussed in https://github.com/bbatsov/rails-style-guide/issues/44#issuecomment-77745495

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,6 +93,14 @@ programming resources.
   `production` one.
 <sup>[[link](#staging-like-prod)]</sup>
 
+* Keep any additional configuration in YAML files under the `config/` directory.
```
Comment (bbatsov) [inline README.md]: You forgot to make this linkable.

File: README.md [hunk 2]
```diff
@@ -93,6 +93,14 @@ programming resources.
...
+
+Since Rails 4.2 YAML configuration files can be easily loaded with the new `config_for` method:
```
Comment (bbatsov) [inline README.md]: This line and the following snippet are not properly indented with respect to the bullet point.

**Comment by vestimir** [comment]
Updated :)

**Comment by bbatsov** [comment]
:+1: Thanks!

**Pull request merged**